packages/experimental-loader/src/cached-stylable-factory.ts

```diff
@@ -1,5 +1,5 @@
 import { Stylable, StylableConfig } from '@stylable/core';
-import { Compiler } from 'webpack';
+import type { Compiler } from 'webpack';
```

Comment:
"importsNotUsedAsValues": "error" should be in tsconfig of `next`.

Reply:
true....and....?

Reply:
do you think we should create the next and merge it here or the opposite?

Reply:
At this point, this branch already consists of all the changes planned for next. we want to enable the flag after the merge to master, but before official release.